src/matcha_ml/cli/ui/functions.py

```diff
@@ -0,0 +1,32 @@
+"""UI functions."""
```

Comment:
Nit: Having a `functions.py` feels slightly too ambiguous - could this be renamed to something more descriptive?

Nit: "UI functions" doesn't really tell me much, what type of functions can be found here?

Reply:
Updated the docstring and filenames to reflect that the module is specifically for user approval functions.